lib/collections/addon/components/discover-page/facets/school-type/component.ts

```diff
@@ -0,0 +1,13 @@
+import Collection from 'ember-osf-web/models/collection';
+import SearchFacetChecklist from '../checklist/component';
+
+class SchoolType extends SearchFacetChecklist {
+    get modelAttribute(): keyof Collection {
```

Comment:
TS can infer the correct return type so not needed.
```suggestion
    get modelAttribute() {
```